Project/Assets/ML-Agents/Examples/Soccer/Scripts/AgentSoccer.cs

```diff
@@ -107,8 +140,22 @@ public void MoveAgent(float[] act)
 
     public override void OnActionReceived(float[] vectorAction)
     {
-        // Existential penalty for strikers.
-        AddReward(-1f / 3000f);
+
+        if (position == Position.Goalie)
+        {
+            // Existential bonus for Goalies.
+            AddReward(1f / 3000f);
```

Comment:
Make a const rather than copy `1f / 3000f` in these 3 places.

Reply:
Also, is this number dependent on the agent max_step ?

Reply:
I added a private float that's set in Initialize using maxStep.